### ISSUE TITLE
App showing incorrect shows in library

### ISSUE THREAD

**Issue by WeebNetsu** [Drumber/Kitsune#3]
Recently the app has been showing that I'm watching Castlevania season 2, however I finished it and have add it to my completed list... Here is how my watch list looks on the website, and you see it only once, Castlevania (season 3):

![Selection_165](https://user-images.githubusercontent.com/52031254/201460119-3b4cd21a-0032-4a91-a108-41cba16ef422.png)

And here is how it looks in the app:

![image](https://user-images.githubusercontent.com/52031254/201460144-6480e45b-6abe-42aa-ba31-be161917649d.png)

And as you can see here, it is in my completed section on Kitsu:

![image](https://user-images.githubusercontent.com/52031254/201460189-9598b651-7997-4ef1-9151-31e21ac0a38c.png)

Here is my Kitsu profile, in case you need it for debugging: https://kitsu.io/users/Netsu

**Comment by Drumber** [comment]
Thanks for reporting and providing detailed info.

The library data returned from the Kitsu API is correct ([here](https://kitsu.io/api/edge/library-entries?filter[id]=90182863&include=anime)). Normally Kitsune should retrieve the latest data from the Kitsu API when you open the library page. It might be that the synchronization failed and therefore the old data from the cache is still displayed.

You could try to do a forced library sync by dragging down in the library page or clear the app cache.

If this doesn't fix the problem, I'll have to investigate further.

**Comment by WeebNetsu** [comment]
I got it fixed, however I had to completely clear all the app data, since clearing cache and refreshing didn't work (basically, I had to log back into my account for it to sync back up correctly again)

However, since this might be a problematic future bug that could potentially be fixed in the future, this was the last screenshot I took of the app before clearing the data, basically, all the castlevanias didn't automatically set as completed in the app if it has been set to completed on the Kitsu website. It did still pick up when I added or updated shows, so just did not update some shows if it has been completed, and set so on the website and not the app

![image](https://user-images.githubusercontent.com/52031254/202904133-22d7bb9f-bde0-476b-b56a-f71477210409.png)

If I run into the issue again, I'll send another screenshot with any new findings

**Comment by Drumber** [comment]
Thanks again for the detailed description of the problem. I could reproduce the bug and I've already published a fix for it.

Some details to the bug:
The problem occurred only when a library filter is selected (e.g. 'Watching'). In case the status of a library entry was changed from outside (Kitsu.io), only the new entries of the selected filter were updated, but old entries were not removed from the local database.
